lib/tasks/rails_db_views_tasks.rake

```diff
@@ -25,28 +25,17 @@ namespace :db do
 
     creator.drop_views
   end
+end
 
-  task :migrate => :environment do
-    Rake::Task['db:create_views'].invoke
-  end
-
-  task :rollback => :environment do
-    Rake::Task['db:create_views'].invoke
-  end
+# prepend "db" namespace
```

Comment:
The original gem still uses the (flawed) rake-hooks gem, which we've (correctly) gotten away from here, plus there are a bunch of updates since this fork was created: https://github.com/anykeyh/rails_db_views/blob/master/lib/tasks/rails_db_views_tasks.rake

So my take is that we've _already_ strayed from the original far enough that our path forward is polarized:

1. We stop using it entirely (eventually) -- seems like the most likely outcome?
2. We maintain our own fork of it until we can actually submit PRs of our changes to the original.

(When I made the changes here, I did so with #2 in mind.)